### ISSUE TITLE
[enh] Yii2 - Lacking grab* functions

### ISSUE THREAD

**Issue by slinstj** [Codeception/Codeception#5044]
Using Yii2 Module we don't have functions like `grabColumnFromDatabase()`, `grabFromDatabase()` and `grabNumRecords()`. It would be great if it could have them.

**Comment by samdark** [comment]
@slinstj are they implemented in any other modules?

**Comment by slinstj** [comment]
Yeah, Db Module. I tryed to use but I had some connection problems with no much time to try solve them so I gave up for now.
